app/controllers/users/sessions_controller.rb

```diff
@@ -11,5 +11,6 @@ def redirect_if_university
     service = OregonDigital::UserAttributeService.new(params[:user])
     redirect_path = service.email_redirect_path
     redirect_to redirect_path unless redirect_path.nil?
```

Comment:
Does this still redirect institutional users without admin, depositor, or collection manager roles to the dashboard?

Reply:
oh good question. ill check

Reply:
Answer: Yes, logging in via institutional login without admin depositor or collection manager still redirects to dashboard

Reply:
I think we want only those three roles to hit the dashboard, everyone else should go to the homepage or the page they left from (if easy)

Reply:
ahhh ok i think the title to the ticket might have misled me. Ill swing back around and verify that everyone except admin/collectionmanagers/depositors route to homepage. Else go to dashboard.